kitsune/products/views.py

```diff
@@ -43,12 +44,17 @@ def product_landing(request, template, slug):
         else:
             latest_version = 0
 
+    user_agent = request.META.get('HTTP_USER_AGENT', '')
+    browser = get_browser(user_agent)
+    show_fx_download = (product.slug == 'thunderbird' and browser != 'Firefox')
```

Comment:
Yes @mythmon. I have understood it. I did not have Idea about this, thanks for your last comment regarding this. :smile: